mappings/net/minecraft/client/render/entity/feature/FeatureRenderer.mapping

```diff
@@ -1,6 +1,5 @@
 CLASS net/minecraft/class_3856 net/minecraft/client/render/entity/feature/FeatureRenderer
 	METHOD method_31187 render (Lnet/minecraft/class_2267;FFFFFFF)V
-		ARG 5 limbAngle
```

Comment:
Not easy to see the logic behind these changes from Github without having Enigma open, but this is a strange one. I'm interested how you came up with this one.

Reply:
At least args 6 & 7 were incorrect and I'm pretty sure this is as well, but I didn't spend time investigating what the proper name is

Reply:
I'll look at it later if I remember. I usually search around until I find an assignment in a child or super class, even looking at modern sometimes. Understandable that you wouldn't want to go through that much trouble for some args.